### PR TITLE
Add mobile emulation support to test automation framework for nexus 7…

### PR DESCRIPTION
**Pull request by KacperLukasiewiczQA**
… (chrome)

### CODE REVIEW

**Review by MarekPawlowskiQA — CHANGES_REQUESTED** [review]
Please add information to readme that show how to run our new functionality in framework.

File: src/test/java/com/intive/template/TestRunner.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
     @BeforeClass
     public static void setup() {
         switch (System.getProperty("browser", "chrome").toLowerCase()) {
-            case "chrome", "bs_chrome" -> WebDriverManager.chromedriver().setup();
+            case "chrome", "bs_chrome", "chrome_mobile" -> WebDriverManager.chromedriver().setup();
```
Comment (MarekPawlowskiQA) [inline src/test/java/com/intive/template/TestRunner.java:23]: nice you caught this :) Not in the scope of this PR but think how we can make this check. more generic, that every time we add new chrome setup our switch or something difrent will recognised it

**Review by MarekPawlowskiQA — APPROVED** [review]
lgtm :)

**Review by ospatrycja — APPROVED** [review]
Looks good to me

**Review by annaslodkowska — APPROVED** [review]

**Pull request merged**